src/NuGet.Core/NuGet.Commands/RestoreCommand/Logging/TransitiveNoWarnUtils.cs

```diff
@@ -658,17 +627,17 @@ public class DependencyNode : IEquatable<DependencyNode>
             // If a node is a project then it will hold these properties
             public NodeWarningProperties NodeWarningProperties { get; }
 
-            public DependencyNode(string id, bool isProject, HashSet<NuGetLogCode> projectWideNoWarn, Dictionary<string, HashSet<NuGetLogCode>> packageSpecificNoWarn)
+            public DependencyNode(string id, bool isProject, NodeWarningProperties nodeWarningProperties)
```

Comment:
somehow this constructor and the one below got flipped. No functional issues, just the order of them flipped creating a bigger changset.

Reply:
Ah, my apologies - I was responding to PR feedback about not changing the public APIs and must have pasted it back in the wrong order.  Thank you for the review!